api/turing/cluster/spark.go

```diff
@@ -295,3 +296,13 @@ func toMegabyte(request string) (*string, error) {
 	strVal := fmt.Sprintf("%sm", strconv.Itoa(int(inMegaBytes)))
 	return &strVal, nil
 }
+
+func getEnvVars(request *CreateSparkRequest) []apicorev1.EnvVar {
+	envVars := defaultEnvVars
+
+	for _, ev := range request.SparkInfraConfig.APIServerEnvVars {
```

Comment:
This might look strange how `request.SparkInfraConfig` contains the platform-level env var configs, but here's where the platform-level configs get passed to each request https://github.com/caraml-dev/turing/blob/2725429c58740774cbec59ceefb255d0fcf5b94a/api/turing/batch/ensembling/controller.go#L201